src/commands/hosting-site-list.ts

```diff
@@ -12,17 +12,23 @@ const TABLE_HEAD = ["Site", "Default URL", "App ID (if set)"];
 export default new Command("hosting:site:list")
   .description("list Firebase Hosting sites")
   .before(requirePermissions, ["firebasehosting.sites.get"])
-  .action(async (options) => {
-    const projectId = getProjectId(options);
-    const sites = await listSites(projectId);
-    const table = new Table({ head: TABLE_HEAD, style: { head: ["green"] } });
-    for (const site of sites) {
-      const siteId = site.name.split("/").pop();
-      table.push([siteId, site.defaultUrl, site.appId || "--"]);
-    }
+  .action(
+    async (
+      options: any // eslint-disable-line @typescript-eslint/no-explicit-any
+    ): Promise<{ sites: Site[] }> => {
```

Comment:
Why isn't this just `Promise<Site[]>`?

Reply:
I'm following the convention from channels, where it does `return { channels };`.  That results in the return value being an object like this.  Not entirely sure why it's like that, but consistency is good